core/src/test/scala/org/ensime/indexer/FileWatcherSpec.scala

```diff
@@ -131,7 +157,7 @@ abstract class FileWatcherSpec extends EnsimeSpec
       }
     }
 
-  it should "detect removed parent base directory" taggedAs (Retryable) in
+  it should "detect removed parent base directory" in
```

Comment:
you can hardcode the wait times here instead of using the defaults, which are perhaps excessive. I think our CI boxes are more reliable now so we could probably reduce the timeouts generally.